fastlane.gemspec

```diff
@@ -111,6 +111,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency('emoji_regex', '>= 0.1', '< 2.0') # Used to scan for Emoji in the changelog
 
+  spec.add_dependency('aws-sdk', '~> 2.3') # Used in match for S3 storage
```

Comment:
Why was `v2` of the AWS SDK used here? Was there something preventing the use of `v3`?

Reply:
Already fixed in https://github.com/fastlane/fastlane/pull/16143.